modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -117,6 +117,13 @@ All page names are validated:
 NOTE: Terms and conditions are only available for API assets. Users are prompted to accept the terms and conditions when requesting API access.
 
 
+=== API Groups
+
+If any of the APIs in an API group have a terms and conditions page, then the API group has a terms and conditions page that cannot be deleted and shows all the terms and conditions of the underlying APIs.
```

Comment:
Suggested wording for brevity: "If any API in an API group has a terms and conditions page...."
I think you need "that" before "shows."
Suggestion: remove "all."

Reply:
Good points. Made all three changes. Thank you!